release/cli/pkg/operations/bundle_release.go

```diff
@@ -110,6 +112,22 @@ func GenerateImageDigestsTable(r *releasetypes.ReleaseConfig) (map[string]string
 	return imageDigests, nil
 }
 
+func SignImagesNotation(r *releasetypes.ReleaseConfig, imageDigests map[string]string) error {
+	releaseRegistryUsername := r.ReleaseClients.ECRPublic.AuthConfig.Username
+	releaseRegistryPassword := r.ReleaseClients.ECRPublic.AuthConfig.Password
+	for _, image := range imageDigests {
```

Comment:
`range` operator on a map returns `<key, value>` instead of `<index, value>`, so you'll need to change this accordingly.